doc/SETUP.md

```diff
@@ -42,6 +42,29 @@
 
 To connect to a remote development LND node you can use a [test account](https://github.com/bumi/lightning-browser-extension/wiki/Test-setup)
 
+## Install and debug extension in Firefox Android
+
+a. **Prerequisite**
+
+1. Install or update [web-ext](https://github.com/mozilla/web-ext) to version 4.1.0 or later.
+2. Install the [Android Platform Tools](https://developer.android.com/tools/releases/platform-tools) (you can use Android studio's [sdk manager](https://developer.android.com/studio/intro/update.html#sdk-manager) or [sdk manager](https://developer.android.com/tools/sdkmanager) command line tools to install android platform tools)
+3. Make sure you have adb installed and in your PATH.
+
+b. **Setup Android Emulator**
+
+1.  Install [Firefox for Android Nightly](https://play.google.com/store/apps/details?id=org.mozilla.fenix) on Android
+2.  Enable [Android USB debugging](https://developer.android.com/studio/debug/dev-options) on the device.
+3.  Attach your device to the development computer using a USB cable. When prompted, allow USB debugging for the connection.
```

Comment:
step b above says use an emulator but these instructions above are for a real device, right?

Reply:
changed that to device